prebid-mobile/modules/rendering/ios-sdk-integration-pb.md

```diff
@@ -10,16 +10,72 @@ sidebarType: 2
 # Custom Bidding Integration
 {:.no_toc}
 
-## Mobile API
-
-The integration and usage of the Rendering API are similar to any other Ad SDK. It sends the bid requests to the Prebid Server and renders the winning bid. 
-
-![In-App Bidding with Prebid](/assets/images/prebid-mobile/modules/rendering/Prebid-In-App-Bidding-Overview-Pure-Prebid.png)
+You can use Prebid SDK to monetize your app with a custom ad server or even without it. Use the `Transport API` to obtain the targeting keywords for following usage with the custom ad server. Use the `Rendering API` to display the winning bid without primary ad server and its SDK.
 
 * TOC
 {:toc}
 
-## Banner API
+## Transport API
+
+The default ad server for Prebid's Mobile SDK is GAM. The SDK can be expanded to include support for 3rd party ad servers through the fetchDemand function. This function returns the Prebid Server bidder key/values (targeting keys), which can then be passed to the ad server of choice. 
+
+In this mode, the publisher will be responsible for the following actions:
+
+* Call fetchDemand with extended targetingDict callback
+* Retrieve targeting keys from extended fetchDemand function
+* Convert targeting keys into the format for your ad server
+* Pass converted keys to your ad server
+* Render ad with Prebid Universal Creative or custom renderer
+
+
+This fuction is accesible for the following ad formats:
```

Comment:
By "this function", do you mean `fetchDemand()`, or do you mean "Targeting and Rendering works for all of the ad formats"

Reply:
yes -  fetchDemand(). I've changed the sentence to 

`This approach is available for the following ad formats:`